about-page.md

```diff
@@ -0,0 +1,47 @@
+# Draft - Karrot's About page
+
+### 1st Part
+[One paragraph of text of what Karrot is about, why it exists, to whom and its vision...]
+
+Karrot exists for groups of people who want to save things from being wasted and share them instead, allowing these groups to organise efficiently, on a local and voluntary basis. Things such food, bikes, clothes or whatever is being excessively thrown away or unused in the context of a consumerist society. 
+
+Our vision, as the team of passionate volunteers developing Karrot, is to make a free and open-source software that enables a movement to grow, a movement that is based on communities and networks that save and share resources and that are socially and ecologically sustainable. Karrot has started as a foodsaving tool (hence the name) and is progressively being developed to become a platform for like-minded initiatives to connect and grow.
```

Comment:
Proposal to change last phrase: "Karrot has started as a foodsaving tool but is widening in scope as it is progressively being developed to become a platform for all kinds of grassroots initiatives to connect and grow."

And maybe add: "Our main motivation is to support physical activities with a layer of digital infrastructure so that our users can concentrate on their sustainability endeavors without getting lost in chaos or bureaucracy."

Reply:
'Physical activities' sounds weird though... Any suggestions on how to phrase this better?

Reply:
Yep, sounds like everybody's going together to the gym :laughing: Maybe "face-to-face activities"? I'd also add "...so that users can concentrate on and scale their sustainability endeavors..." if you think it won't break the flow of reading by adding too much wording.